app/src/test/java/edu/uc/kansakpb/ikeahjalpare3/FurnitureDataUnitTest.kt

```diff
@@ -24,11 +24,16 @@ class FurnitureDataUnitTest {
     var rule: TestRule = InstantTaskExecutorRule()
     lateinit var mvm: MainViewModel
 
-    var furnitureService = mockk<FurnitureService>()
+    private var furnitureService = mockk<FurnitureService>()
 
     @Test
     fun confirmKopangDrawerDresser_outputsKopangDrawerDresser () {
-        var furniture: Furniture = Furniture("Kopang Lamp", "https://www.ikea.com/us/en/assembly_instructions/koppang-drawer-dresser__AA-2066877-1_pub.pdf", "https://www.youtube.com/watch?v=biL6K6OESe4")
+        val furniture = Furniture(
```

Comment:
Furniture constructor requires 5 parameters. Tests failed to compile due to missing parameters when instantiating Furniture objects.